test/test_jit.py

```diff
@@ -70,5 +70,16 @@ def __call__(self, b:Tensor) -> Tensor:
       c = fun(b)
       np.testing.assert_equal(c.numpy(), fun.a.numpy()+b.numpy())
 
+  def test_shape_change_jit(self):
```

Comment:
I will merge just this test with an expected failure for now if you want to submit it.